doc/source/config.rst

```diff
@@ -5,62 +5,105 @@ Configuration
 Bandit Settings
 ---------------
 
-Projects may include an INI file named `.bandit` that specifies command line
-arguments that should be supplied for that project. The currently supported
-arguments are:
+Projects may include an INI file named *.bandit*, or a YAML or TOML file
+that needs to be explicitly specified using the ``-c`` option, which specifies
```

Comment:
The INI file is for command line arguments. The YAML file is for configuration such as defining new profiles. This statement might confuse folks to believe all three do the same job, which they don't.